$NetBSD: patch-aa,v 1.14 2011/02/24 13:36:57 adam Exp $

Do not check compiler version. Clang impersonates GCC and might get caught.
Do not treat warnings as errors.

--- configure.ac.orig	2010-09-24 17:37:41.000000000 +0000
+++ configure.ac
@@ -64,10 +64,6 @@ parts=`echo $GCC_VERSION | tr '.' ' '`
 set $parts
 GCC_MAJOR_VERSION=$1
 GCC_MINOR_VERSION=$2
-if test ${GCC_MAJOR_VERSION}${GCC_MINOR_VERSION} -lt 33 
-then
-	AC_MSG_ERROR([*** G++ major version $GCC_VERSION is too old.])
-fi
 
 #######################################################################
 # Configuration options                                               #
@@ -111,7 +107,7 @@ then
 #	CFLAGS="-Werror -Wno-unused -Wno-sign-compare $CFLAGS"
 
 	# The current networking code breaks strict aliasing in g++ 4.5.
-	CXXFLAGS="-Werror -Wno-unused-parameter -Wno-strict-aliasing $CXXFLAGS"
+	CXXFLAGS="-Wno-unused-parameter -Wno-strict-aliasing $CXXFLAGS"
 fi
 
 # Make tests default in svn version
